acestep/ui/gradio/interfaces/training_dataset_tab_label_preview.py

```
@@ -4,13 +4,17 @@
 
 import gradio as gr
 
+from acestep.ui.gradio.direction_utils import auto_heading_id, auto_textbox_id # noqa: F401 – used once placeholders are i18n-ified 
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)